Assets/_Game/Scripts/UI/UICredits.cs

```diff
@@ -12,6 +12,6 @@ public void Setup(GameSystems systems)
 
     public void OnBackClicked()
     {
-        _systems.StateManager.NavigateToState(null, true);
+        _systems.StateManager.NavigateToState(typeof(MainMenuState), true);
```

Comment:
This will lead to multiple mainmenustates in the queue. Better fix would be to do 2 transitions rather than 1 when you push the credits after the game. One to pop the current state (the game state) and transfer it back to main menu and then push the CreditState right after